app-crypt/paperkey/paperkey-1.4.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Tool to extract secret bits from a secret key to be printed on paper"
+DESCRIPTION="Paperkey extracts the secret bits from OpenPGP keys, to be printed \
+on paper as a form of backup and allows it to be reintroduced to reconstruct the \
+secret key if needed."
+HOMEPAGE="http://www.jabberwocky.com/software/paperkey/"
+COPYRIGHT="2007-2016 David Shaw"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.jabberwocky.com/software/paperkey/paperkey-$portVersion.tar.gz"
+CHECKSUM_SHA256="e12bb0ec835127d12a922a8d60b3dfdb3ca8ee60bb5b4d15ae4cea85bbcf336f"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	paperkey$secondaryArchSuffix = $portVersion
+	cmd:paperkey$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	REQUIRES="$REQUIRES
+		lib:libcrypto$secondaryArchSuffix
+		"
+fi
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	BUILD_REQUIRES="$BUILD_REQUIRES
+		devel:libcrypto$secondaryArchSuffix
+		"
+fi
+BUILD_PREREQUIRES="
```

Comment:
Shall we add a blank line before BUILD_PREREQUIRES ?
There isn't any official rule for it, but we usually have one such blank line between the last line of BUILD_REQUIRES and BUILD_PREREQUIRES **_when_** the last part of BUILD_REQUIRES is a conditional block (i.e. an if then .. fi).
(Sorry, I forgot to mention this in my previous review.)